health/micro-ui/web/micro-ui-internals/packages/modules/health-dss/src/pages/employee/L1Main.js

```
@@ -0,0 +1,698 @@
+                          : item.name.includes("PROJECT_STAUS") || item.name.includes("LIVE_ACTIVE_ULBS")
+                          ? "h-100"
+                          : ""
+                      }`}
+                      style={
+                        item.vizType == "collection" || item.name.includes("PROJECT_STAUS") || item.name.includes("LIVE_ACTIVE_ULBS")
+                          ? { backgroundColor: "#fff" }
+                          : { backgroundColor: colors[index].light, padding: "20px" }
+                      }
+                      key={index}
+                      onClick={() => routeTo(`/${window.contextPath}/employee/dss/dashboard/${item.ref.url}`)}
+                    >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)